Renamed add bookmark for better understanding

Mash707 opened the pull request:
Resolves #657 


![image](https://github.com/user-attachments/assets/5227bd0c-bcbd-49ea-bc09-6e723132b5e8)
![image](https://github.com/user-attachments/assets/088d65f1-52d4-4b16-9bf5-db95d418272f)

Mash707 commented:
@garrett could you review the changes. Thank you.

martinpitt commented on po/LINGUAS:
```diff
@@ -0,0 +1,11 @@
+cs
```
Please drop this file from the commit, it doesn't belong here.

Mash707 replied:
Do I need to delete the whole file?

martinpitt commented on src/files-breadcrumbs.tsx:
```diff
@@ -142,9 +142,9 @@ function BookmarkButton({ path }: { path: string }) {
     if (!defaultBookmarks.some(bkmark => bkmark.loc === path)) {
         if (bookmarks.includes(path)) {
-            actionText = _("Remove current directory");
+            actionText = _("Remove from bookmarks");
```
This renaming also needs to be reflected in the tests:

```
❱❱❱ git grep 'Remove current' test
test/check-application:        b.wait_not_present(".pf-v5-c-menu .pf-v5-c-menu__list-item:contains(Remove current directory)")
test/check-application:        b.click(".pf-v5-c-menu .pf-v5-c-menu__list-item:contains('Remove current directory') button")
test/check-application:        b.click(".pf-v5-c-menu .pf-v5-c-menu__list-item:contains('Remove current directory') button")
test/check-application:        b.click(".pf-v5-c-menu .pf-v5-c-menu__list-item:contains('Remove current directory') button")
test/check-application:        b.click(".pf-v5-c-menu .pf-v5-c-menu__list-item:contains('Remove current directory') button")
test/check-application:        b.click(".pf-v5-c-menu .pf-v5-c-menu__list-item:contains('Remove current directory') button")
```

martinpitt commented on src/files-breadcrumbs.tsx:
```diff
@@ -142,9 +142,9 @@ function BookmarkButton({ path }: { path: string }) {
+            actionText = _("Remove from bookmarks");
         } else if (cwdInfo !== null) {
-            actionText = _("Add bookmark");
+            actionText = _("Add to bookmarks");
```
Same here.
```
❱❱❱ git grep 'Add bookmark' test
test/check-application:        b.wait_not_present(".pf-v5-c-menu .pf-v5-c-menu__list-item:contains(Add bookmark)")
test/check-application:        b.click(".pf-v5-c-menu .pf-v5-c-menu__list-item:contains(Add bookmark) button")
test/check-application:        b.click(".pf-v5-c-menu .pf-v5-c-menu__list-item:contains(Add bookmark) button")
test/check-application:        b.click(".pf-v5-c-menu .pf-v5-c-menu__list-item:contains(Add bookmark) button")
test/check-application:        b.click(".pf-v5-c-menu .pf-v5-c-menu__list-item:contains(Add bookmark) button")
test/check-application:        b.wait_not_present(".pf-v5-c-menu .pf-v5-c-menu__list-item:contains('Add bookmark') button")
test/check-application:        b.click(".pf-v5-c-menu .pf-v5-c-menu__list-item:contains(Add bookmark) button")
test/check-application:        b.click(".pf-v5-c-menu .pf-v5-c-menu__list-item:contains(Add bookmark) button")
```

martinpitt reviewed (changes requested):
Thank you for fixing this! Needs a few cleanups though.

martinpitt commented:
Triggered the tests. Can you please squash the fixups and remove the "Signed-of-By"? We don't do that in cockpit. Thanks!

Mash707 commented:
> Triggered the tests. Can you please squash the fixups and remove the "Signed-of-By"? We don't do that in cockpit. Thanks!

I have squashed the commits and also removed "Signed-of-By". Could you tell if it is done properly? I don't have much experience with squashing commits😅😅. Thank you.

martinpitt reviewed (approved):
Excellent, thank you @Mash707 !
